runtime/hydradx/src/lib.rs

```diff
@@ -1135,12 +1158,142 @@ impl_runtime_apis! {
 
 				fn get_asset() -> Asset {
 					Asset {
-						id: AssetId(Location::here()),
+						id: AssetId(PolkadotLocation::get()),
 						fun: Fungible(ExistentialDeposit::get()),
 					}
 				}
 			}
 
+			use primitives::constants::currency::UNITS;
```

Comment:
we should probably split this to separate file as it's getting crowded

Reply:
you remember, the last time when we tried to put the runtime apis into a separate file, we had a weird error on prod, so maybe it is just better to leave it in lib.rs...